Tools/Scripts/webkitpy/benchmark_runner/benchmark_runner.py

```diff
@@ -167,7 +170,7 @@ def _run_benchmark(self, count, web_root):
 
                         results.append(run_result)
                     else:
-                        raise Exception('Plan does not contain entry_point or test_files')
+                        raise Exception('Plan does not contain entry_point/subtest_entry_point or test_files')
 
```

Comment:
This doesn't seem to accurately describe the situation.

Reply:
How so? We hit this line if the plan contains no entry points (either the default entry_point or the new subtest_entry_point), or if no files are specified. Is there a way we can make this clearer?

Reply:
We only hit this else condition when `'entry_point' in self._plan` is false.